library/system.lisp

```diff
@@ -76,55 +76,56 @@ While the result will always contain microseconds, some implementations may retu
 
   (declare get-run-time (Unit -> UFix))
   (define (get-run-time)
-    "Gets the run-time."
+    "Gets the run-time in microseconds."
     (lisp UFix ()
       (cl:get-internal-run-time)))
```

Comment:
this is not guaranteed to be microseconds. you just use a library or `internal-time-units-per-second`